tests/test_sigmaJs.py

```diff
@@ -381,21 +376,17 @@ def test_str_to_bool():
 
     assert parser._str_to_bool('true')
     assert not parser._str_to_bool('false')
-    with pytest.raises(ValueError) as e:
+    with pytest.raises(Exception, match="err- negation is not bool"):
         assert parser._str_to_bool('error')
-    assert str(
-        e.value) == 'err- negation is not bool'
 
 
 def test_use_bat_report_file():
     src = ('test_data/xccdf_org.ssgproject.content_rule_sssd_' +
            'ssh_known_hosts_timeout-comment.fail.sh-xccdf_or' +
            'g.ssgproject.content_profile_ospp-results-initial.xml')
 
-    with pytest.raises(ValueError) as e:
-        parser = tests.any_test_help.get_parser(src)
-    assert str(
-        e.value) == 'err- This is not arf report file.'
+    with pytest.raises(Exception, match="err- This is not arf report file."):
```

Comment:
Similar with the message here, I would check that either `arf` or `ARF` are present in the message string. You can use `\b` in those regexes, as they match the word boundary, so e.g. the word `arford` doesn't match.